### PR TITLE
lerna import intl-messageformat-parser

### PR DESCRIPTION
**Pull request by longlho**
Let's see if the CLI works

### CODE REVIEW

File: packages/intl-messageformat-parser/package.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    "plural",
+    "icu"
+  ],
+  "author": "Eric Ferraiuolo <eferraiuolo@gmail.com>",
+  "license": "BSD-3-Clause",
+  "bugs": {
+    "url": "https://github.com/yahoo/intl-messageformat-parser/issues"
+  },
+  "homepage": "https://github.com/yahoo/intl-messageformat-parser",
+  "devDependencies": {
```
Comment (redonkulus) [inline packages/intl-messageformat-parser/package.json]: Just need to move these to the monorepo

**Review by redonkulus — CHANGES_REQUESTED** [review]

File: packages/intl-messageformat-parser/package.json (hunk 1 above)
Comment (longlho) [reply packages/intl-messageformat-parser/package.json]: done

File: package.json [hunk 2]
```diff
@@ -18,12 +18,20 @@
     "lint": "eslint .",
     "test": "lerna run lint --since --stream && lerna run test --since --stream"
   },
-  "dependencies": {},
+  "dependencies": {
+    "intl-messageformat-parser": "file:packages/intl-messageformat-parser"
```
Comment (redonkulus) [inline package.json:22]: Did lerna do this automatically?

**Review by redonkulus — APPROVED** [review]

**Comment by longlho** [comment]
Looks like it does yeah

File: package.json (hunk 2 above)
Comment (longlho) [reply package.json:22]: yeah it did

**Pull request merged**